Reach/Payment/Model/ResourceModel/Carrier/CsvHsCode/CSV/RowParser.php

```diff
@@ -82,8 +103,31 @@ private function getHsCode(array $rowData, ColumnResolver $columnResolver)
     {
         $hsCode = $columnResolver->getColumnValue(ColumnResolver::COLUMN_HSCODE, $rowData);
         if ($hsCode === '') {
-            $hsCode = '*';
+            $hsCode = null;
         }
         return $hsCode;
     }
+
+    /**
+     * @param array $rowData
+     * @param ColumnResolver $columnResolver
+     * @return int|string
+     * @throws ColumnNotFoundException
+     */
+    private function getCountryOfOrigin(array $rowData, ColumnResolver $columnResolver)
+    {
+        $countryOfOrigin = $columnResolver->getColumnValue(ColumnResolver::COLUMN_COUNTRYOFORIGIN, $rowData);
+        if (strlen($countryOfOrigin) != 2) {
+            if(strlen($countryOfOrigin) > 0){
+                $this->_logger->debug('----------------IMPORT ERROR----------------');
+                $sku = $columnResolver->getColumnValue(ColumnResolver::COLUMN_SKU, $rowData);
+                $this->_logger->debug("Country of origin for sku ".$sku." was inputted incorrectly: ".$countryOfOrigin);
```

Comment:
please indicate what it should have been in the debug message (for example, "...was entered incorrectly: {countryOfOrigin} expecting a two character ISO country code")